test/javascript/components/student/Editor.test.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { render, fireEvent, waitFor } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import { rest } from 'msw'
+import { setupServer } from 'msw/node'
+import { Editor } from '../../../../app/javascript/components/student/Editor'
+
+const server = setupServer(
+  rest.post('https://exercism.test/submissions', (req, res, ctx) => {
+    return res(
+      ctx.json({ id: 2, tests_status: 'pending', test_runs: [], message: '' })
+    )
+  })
+)
+
+beforeAll(() => server.listen())
+afterEach(() => server.resetHandlers())
+afterAll(() => server.close())
+
+test('clears current submission when resubmitting', async () => {
```

Comment:
Are we testing that the current submission is cleared or that a new submission is pending?

Reply:
We're testing that the previous submission was cleared. How can we rewrite this test better?

Reply:
Oh I was not commenting on the actual test, but only on the name of the test :) Maybe something like:

```suggestion
test('resubmitting creates pending submission', async () => {
```

or something like that